exasol/python_extension_common/deployment/temp_schema.py

```diff
@@ -0,0 +1,41 @@
+import pyexasol     # type: ignore
```

Comment:
Again, I don't like this refactoring because it impacts the projects that use the `temp_schema`.

Reply:
I was unaware that `temp_schema` is part of an externally used interface. 
Thanks again, for pointing this out!

Reply:
I see the following options 
* O1) 
  * rename file `extract_validator.py` to `language_container_validator.py` 
  * move function `temp_schema` back into this file 
  * delete file `temp_schema.py`
* O2) 
  * Create a file `language_container_validator.py` 
  * import  `temp_schema` 
  * Add a comment "# This file is only for providing backwards compatibility "
* O3) 
  * keep like it is, breaking the backward compatibility

My favorite would be O3 due to the file names begin shorter, prefix-free and more precise IMHO.
But I could also live with O2, hoping to remove the additional file someday.

Reply:
O1 or O2 please, but I think we could also add `import temp_schema` in the `__init__.py`. I wish we had done this from the start, then we would be free to move it around as we liked.

Reply:
Then O2 - deal :handshake:

Reply:
Done